autoemulate/experimental/model_selection.py

```diff
@@ -84,11 +83,11 @@ def cross_validate(  # noqa: PLR0913
     cv: BaseCrossValidator,
     dataset: Dataset,
     model: type[Emulator],
+    model_config: ModelConfig,
```

Comment:
Just a suggestion to avoid having to pass it if the user just wants to use the default? I can also see an argument against this as opening up the potential for user error.

```suggestion
    model_config: ModelConfig = {},
```